manager/integration/tests/test_recurring_job.py

```diff
@@ -483,3 +483,39 @@ def test_recurring_jobs_for_detached_volume():  # NOQA
     17. cleanup
     """
     pass
+
+
+@pytest.mark.skip(reason="TODO")
+def test_recurring_jobs_on_nodes_with_taints():  # NOQA
+    """
+    Test recurring jobs on nodes with taints
+
+    Context:
+
+    longhorn/longhorn#1903
+
+    Steps:
+
+    1. Set taint toleration for Longhorn components
+       `persistence=true:NoExecute`
+    1. Taint `node-1` with `persistence=true:NoExecute`
+    1. Create a volume, vol-1.
+       Attach vol-1 to node-1
+       Write some data to vol-1
+    1. Create a recurring backup job which runs every minute
+    1. Wait for 3 minutes.
+       Verify that the there is 1 backup created
+       Verify that the total number of pod in longhorn-system namespace < 50
+       Verify that the number of pods of the cronjob is <= 2
+
+    1. Taint all nodes with `persistence=true:NoExecute`
+    1. Write some data to vol-1
+    1. Wait for 3 minutes.
+       Verify that the there are 2 backups created in total
+       Verify that the total number of pod in longhorn-system namespace < 50
+       Verify that the number of pods of the cronjob is <= 2
+
+    1. Remove `persistence=true:NoExecute` from all nodes and Longhorn setting
+       Clean up backups, volumes
+    """
+    pass
```

Comment:
I think there is a new line required here or you can remove the ```pass``` as the test is already skipped.

Reply:
Thank you! Fixed